### DISCUSSION TITLE
Optimising VS Code for Mobile

### DISCUSSION THREAD

**Discussion by Splizard** [Splizard/vscode-mobile#1] [Announcements]
This is a home for discussing ideas on how to optimise VS Code to have a better user experience on mobile touch devices. If you have ideas or suggestions,feel free to open a new discussion. Please keep activity within this repo mobile-specific, general VS Code issues should be reported on the upstream Microsoft repository.

**Comment by zongou** [comment]
I have fixed keyboard popping up when scrolling on android, pull request will come soon

**Reply by zongou** [reply]
i will commit to my own branch for my own optimization
https://github.com/zongou/vscode
and there are prebuilts in github actions you can test

**Comment by Splizard** [comment]
Cheers @zongou if you are referring to the PR you made to the Microsoft repo, I've already merged it into main.
https://github.com/Splizard/vscode-mobile/commit/e4197cd86249934979cec13e0f23e5188db51195

I'm still trying to sort out how to complete an android build, so far I've only tested my pen-changes by connecting via mobile to a remote code-server. Do you have a build process for Android ie. Termux or something else? I spent some time last night trying to build on my phone but I threw half an hour of compilation at vs code without any result.

`yarn build:vscode` (termux) 

I also had to patch process.system or something to trick playwright and electron into thinking I'm running 'linux' instead of unsupported operating system 'android'. I'll probably reach out to the code-server guys and try to pin down what their build process actually is.

**Reply by zongou** [reply]
Ok I have noticed the PR thanks for noticing.

you can checkout my this github action to build for android

https://github.com/zongou/actions/blob/main/.github/workflows/build_vscode_server.yaml

it is build on top of aarch64-linux-gnu version, only three node modules are replaced:

* spdlog
* node-pty
* @parcel/watcher

I will update this github action to be able to build specific repo/commit soon in order to build for this repo
